client/src/components/ReviewsList.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Review from './review';
+
+class ReviewsList extends React.Component {
+  constructor(props) {
```

Comment:
Good separation of concerns here having the review list separated from the actual review component